src/diffusers/schedulers/scheduling_utils.py

```diff
@@ -54,6 +54,9 @@ def log(self, tensor):
 
         raise ValueError(f"`self.tensor_format`: {self.tensor_format} is not valid.")
 
+    def sqrt(self, tensor):
+        return tensor**0.5
```

Comment:
Let's avoid one line functions :)

Reply:
+1 also `** 0.5` works for all frameworks